### PR TITLE
add calour recipe

### PR DESCRIPTION
**Pull request by RNAer**

### CODE REVIEW

File: recipes/calour/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+{% set sha256 = "09b62a876e9bf32db996589362219b622feb0ca7b3a0739531e80fb2ae2b9246" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
```
Comment (synapticarbors) [inline recipes/calour/meta.yaml]: Please delete comments

File: recipes/calour/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - pandas
+    - matplotlib >=2.0.0
+    - scikit-learn >=0.19.1
+    - scikit-bio >=0.5.2
+    - biom-format
```
Comment (synapticarbors) [inline recipes/calour/meta.yaml]: It looks like you're going to need to have `biom-format` submitted and accepted before you can build this package.

File: recipes/calour/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    exploration of microbiome (and other matrix form data),
+    incorporating external databases.
+
+  doc_url: https://biocore.github.io/calour/
+  dev_url: https://github.com/biocore/calour
+
+extra:
+  recipe-maintainers:
+    - RNAer
+    - amnona
```
Comment (synapticarbors) [inline recipes/calour/meta.yaml]: Can you provide confirmation that this co-maintainer is willing to take on that role?

File: recipes/calour/meta.yaml (hunk 2 above)
Comment (RNAer) [reply recipes/calour/meta.yaml]: it's already there: https://bioconda.github.io/recipes/biom-format/README.html

Comment (synapticarbors) [reply recipes/calour/meta.yaml]: That looks like it's hosted on `bioconda`. My understanding from looking at the config files on conda-forge is that it only looks for packages in `defaults`, `conda-forge` and `local` channels.

Comment (RNAer) [reply recipes/calour/meta.yaml]: ah, sorry. you're right. will do that

**Comment by RNAer** [comment]
@synapticarbors , thanks for the reminder. currently the bottleneck is that the dependency of biom-format is only available in bioconda - is there anyway to get around that?

I am also ok to close this for the time being.

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes @conda-forge/bioconda-recipes What's the policy on how to deal with a dependency that's on bioconda? Normally we'd just package the dependency if it was missing from the `default` and `conda-forge` channels, but it seems like we're trying not to duplicate bioconda recipes. Should this recipe be on bioconda rather than conda-forge?

**Comment by isuruf** [comment]
> Normally we'd just package the dependency if it was missing from the default and conda-forge channels

That's the policy

> but it seems like we're trying not to duplicate bioconda recipes.

No, we are just trying not to have name clashes with bioconda. (Same name for the conda package, but the packages are actually different

> Should this recipe be on bioconda rather than conda-forge?

Probably should be in bioconda, because the dependency is in bioconda.

**Comment by synapticarbors** [comment]
@RNAer -- Would you prefer to have this package in conda-forge, or do you think it would be better suited for bioconda? I'll leave that up to you. If you want to keep it here, then we'll need to package `biom-format` in conda-forge. I can help with that as needed.

**Comment by synapticarbors** [comment]
PR for `biom-format` is at https://github.com/conda-forge/staged-recipes/pull/6338

**Comment by RNAer** [comment]
thanks, @synapticarbors ! I saw the controversies on this and `biom-format` should be on conda-forge and bioconda. I am fine with closing it.

**Pull request closed without merging**